### DISCUSSION TITLE
Way to disable terminal output?

### DISCUSSION THREAD

**Discussion by haldunanil** [withcatai/node-llama-cpp#240] [Q&A] [answered]
Hi 👋  I'm trying to use node-llama-cpp as part of a CLI tool but don't want all the terminal logs to be printed out like so:

```
llama_model_loader: Dumping metadata keys/values. Note: KV overrides do not apply in this output.
llama_model_loader: - kv   0:                       general.architecture str              = llama
...
```

Is there a way to disable this when using the `LlamaModel` class in Node?

**Comment by giladgd** [comment, marked as answer]
I recommend switching to [version 3 beta](https://github.com/withcatai/node-llama-cpp/pull/105), where it only logs warning logs by default, and you can also configure it further if you don't want to see logs at all
